pycheribuild/projects/cherisim.py

```diff
@@ -127,12 +127,21 @@ def __init__(self, config: CheriConfig):
         super().__init__(config)
         self.make_args.set(JTAG_ATLANTIC=1) # MUCH faster
 
+    @classmethod
+    def setup_config_options(cls, **kwargs):
```

Comment:
Could make this a global config option so you don't have to add it to `run-fpga` as well?

Reply:
Would that be visible to other projects as well?

Reply:
You could also do BuildBeriCtl.get_instance().env_script if you don't want to pollute the global options namespace. But since there are three projects that would need it, I think it's fine to make it a global option.

Reply:
BuildCheriSim should also use this setting.

Reply:
Right, I prefer the global option to reading other project's configs. I will also switch BuildCheriSim to use it.